cob_teleop/ros/src/joy_init

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+#
+# Copyright 2017 Fraunhofer Institute for Manufacturing Engineering and Automation (IPA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rospy
+from sensor_msgs.msg import Joy
+from std_srvs.srv import Trigger, TriggerRequest
+
+class JoyInit(object):
+  def __init__(self):
+
+    rospy.wait_for_service("/base/driver/init")
+    self.init_client = rospy.ServiceProxy("/base/driver/init", Trigger)
+    rospy.wait_for_service("/base/driver/recover")
+    self.recover_client = rospy.ServiceProxy("/base/driver/recover", Trigger)
+
+    self.last_init = rospy.Time()
+    self.initialized = False
+    self.joy_subscriber = rospy.Subscriber("/joy", Joy, self.joy_callback)
+
+
+
+  def joy_callback(self, joy_message):
+    if (rospy.Time.now() - self.last_init)  < rospy.Duration(1):
+      rospy.logdebug("still cooldown")
+      return
+    if joy_message.buttons[4] and joy_message.buttons[5] and joy_message.buttons[9]:
```

Comment:
maybe you can at least keep the part where the joy buttons were configurable via param
see https://github.com/ipa320/cob_command_tools/blob/1a01d8641d200bace5ae65c3e5b7f4e05aa031bb/cob_teleop/ros/src/cob_teleop.cpp#L453
at least introduce `get_param` calls with respective default values 4, 5 and 9 - reading from relative namespace